cleverhans/attacks.py

```diff
@@ -490,80 +490,42 @@ def __init__(self, model, back='tf', sess=None):
             raise NotImplementedError(error)
 
         import tensorflow as tf
-        self.feedable_kwargs = {'y_target': tf.float32}
-        self.structural_kwargs = ['theta', 'gamma',
+        self.feedable_kwargs = {'y_target': tf.int32}
+        self.structural_kwargs = ['theta', 'epochs', 'pair', 'min_proba',
```

Comment:
In the rest of the library, `prob` and `probs` have been used to refer to probabilities so it would be nice to keep things consistent here as well.